karate-core/src/main/java/com/intuit/karate/driver/android/AndroidMobileDriver.java

```diff
@@ -0,0 +1,52 @@
+package com.intuit.karate.driver.android;
+
+import com.intuit.karate.FileUtils;
+import com.intuit.karate.Http;
+import com.intuit.karate.Logger;
+import com.intuit.karate.core.ScenarioContext;
+import com.intuit.karate.driver.AppiumMobileDriver;
+import com.intuit.karate.driver.DriverOptions;
+import com.intuit.karate.shell.CommandThread;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * @author babusekaran
+ */
+public class AndroidMobileDriver extends AppiumMobileDriver {

```

Comment:
again - but you can decide, is `AndroidDriver` sufficient

Reply:
`IosDriver`, `AndroidDriver`, `AppiumDriver` looks sufficient. making changes.

Reply:
done https://github.com/intuit/karate/pull/744/commits/0d1ad4303531390eff58e6e2a377dd536b86f79a